chia/wallet/wallet_node.py

```diff
@@ -230,6 +232,9 @@ async def _start(
         if self.state_changed_callback is not None:
             self.wallet_state_manager.set_callback(self.state_changed_callback)
 
+        self.last_wallet_tx_resend_time = int(time.time())
+        if "tx_resend_timeout_secs" in self.config:
+            self.wallet_tx_resend_timeout_secs = self.config["tx_resend_timeout_secs"]
```

Comment:
Confused by this - won't this mean that everyone upgrading will have a timeout of 60 seconds (because they won't have this new option in config.yaml), while new installs have a timeout of 1800s ? That seems unintentional, shouldn't they have the same default?